src/main/java/com/vadmack/mongodbtest/service/ProjectService.java

```diff
@@ -22,22 +24,24 @@
 
     private final ModelMapper modelMapper = new ModelMapper();
 
-    public List<Project> findAll() {
-        return repository.findAll();
+    public List<ProjectDto> findAll() {
+        return repository.findAll().stream().map(project -> modelMapper.map(project, ProjectDto.class))
+                .collect(Collectors.toList());
     }
 
-    public Project findById(Long id) {
-        return repository.findById(id).orElse(new Project());
+    public ProjectDto findById(Long id) {
+        return repository.findById(id).map(project -> modelMapper.map(project, ProjectDto.class))
+                .orElse(new ProjectDto());
     }
 
-    public void create(ProjectDto projectDto) {
-        Project project = modelMapper.map(projectDto, Project.class);
+    public void create(ProjectNoIdDto projectNoIdDto) {
+        Project project = modelMapper.map(projectNoIdDto, Project.class);
```

Comment:
`Project project = modelMapper.map(projectNoIdDto, Project.class);` is repeated several times in the service. I suggest to  create and use new method like this:
`Project dtoToProject(ProjectNoIdDto dto)`